cmd/adsysd/integration_tests/adsysctl_service_test.go

```diff
@@ -333,7 +333,7 @@ func TestServiceStatus(t *testing.T) {
 
 			// Create users krb5cc and GPO caches
 			if !tc.noCacheUsersMachine {
-				krb5UserDir := filepath.Join(adsysDir, "run", "krb5cc")
+				krb5UserDir := filepath.Join(adsysDir, "run", "krb5cc", "tracking")
```

Comment:
should we only create that one and not the ticket copy too to simulate a real run state?